### ISSUE TITLE
Accessiblity Issue with "show # results"

### ISSUE THREAD

**Issue by gg0309** [w3c/wcag#266]
There is an accessibility issue on the WET 4.0 working examples page with the Show # entries code.  The issue is the <label> includes the select:

<label>
           Show
                 <select class="" name="components_length" aria-controls="components">
                         <option value="10">10</option>
                         <option value="25">25</option>
                         <option value="-1">All</option>
                  </select>
            entries
</label>

This causes a failure with Success Criteria 1.3.1, 4.1.2 F68: Failure of Success Criterion 1.3.1 and 4.1.2 due to the association of label and user interface controls not being programmatically determinable.

**Comment by alastc** [comment]
Sorry for the delay, but I'm not sure what you're refering to?

Some of that text is on the [4.1.2 understanding page](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html), and probably the [quickreference](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=412#name-role-value), but I don't know where the Show entries bit comes from?

Could you provide a URL?

**Comment by patrickhlauke** [comment]
i'll assume this has something to do with WET and should likely be filed there as an issue https://github.com/wet-boew/wet-boew

**Comment by patrickhlauke** [comment]
as the original reporter here doesn't seem to have been active in the last 2 years since posting this, and it does indeed sound like it's about WET, i'll be bold and close this. feel free to reopen if it turns out to be relevant.

**Comment by patrickhlauke** [comment]
(@alastc feel free to tell me to stay in my lane if my jumping in like this is harmful rather than helpful)

**Comment by alastc** [comment]
In this case, it's fine, any help with weeding is welcome. Obviously (and explaining for everyone else) if it were a valid public comment then we'd need to go through process.
